cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/get_compute_fleet_status.erb

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+<%= node['cluster']['cookbook_virtualenv_path'] %>/bin/python <%= node['cluster']['scripts_dir'] %>get-compute-fleet-status.py --table-name <%= node['cluster']['ddb_table'] %> --region <%= node['cluster']['region'] %>
```

Comment:
I know that this doesn't accept parameters but you can add an --help flag, which will report a brief description on what the command does

Reply:
Added